packages/aria-voyager/tests/listbox/navigation/focus/no-selection.test.ts

```diff
@@ -3,14 +3,14 @@ import { expect, test } from 'vitest';
 import { Listbox } from '../../../../src';
 import { createListWithFruits, getItems } from '../../-shared';
 
-test('focus activates the first item', () => {
+test('focus activates the first item', async () => {
   const list = createListWithFruits();
   const listbox = new Listbox(list);
   const { firstItem, secondItem } = getItems(listbox);
 
   list.dispatchEvent(new FocusEvent('focusin'));
 
-  expect(list.getAttribute('aria-activedescendant')).toBe(firstItem.id);
-  expect(firstItem.getAttribute('aria-current')).toBe('true');
-  expect(secondItem.getAttribute('aria-current')).toBeNull();
+  await expect.element(list).toHaveAttribute('aria-activedescendant', firstItem.id);
```

Comment:
this looks great. But unfortunately does not combat the present flakiness with the tests.

Can you rewrite the `await expect.element()` with their `await expect.poll()` versions.